client_test.go

```diff
@@ -117,7 +117,7 @@ func TestClientTLSSyncV4(t *testing.T) {
 		Certificates: []tls.Certificate{cert},
 	}
 
-	s, addrstr, err := RunLocalTLSServer(":0", &config)
+	s, addrstr, err := RunLocalTLSServer(":0", &config, time.Second, time.Hour)
```

Comment:
It should be possible to set these timers after the creation of the server? That would cut down on this PR's size as we don't needlessly need to update a bunch of tests.